### DISCUSSION TITLE
赞助之后的快照版登陆微软账号出现了以下问题：

### DISCUSSION THREAD

**Discussion by WonderfulStone** [Meloong-Git/PCL#1903] [提问]
![QQ截图20230527151200](https://github.com/Hex-Dragon/PCL2/assets/134767692/19a03ed3-0579-4b8d-a516-00d2b68fe896)
求解！

**Comment by WonderfulStone** [comment]
微软账号本身我看了是没有问题的，其他启动器也可以登录

**Comment by FangFuJi1991** [comment]
401

**Comment by WorldHim** [comment]
超时关闭，若有需要可继续回复。
